src/org/labkey/test/pages/study/CreateDatasetPage.java

```diff
@@ -17,7 +17,6 @@
 
 import org.labkey.test.Locator;
 import org.labkey.test.components.html.Input;
-import org.labkey.test.pages.EditDatasetDefinitionPage;
 import org.labkey.test.pages.LabKeyPage;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
```

Comment:
I think this `CreateDatasetPage` is for the old designer. Delete it?

Reply:
Looks not to be used anywhere- I think it was there to wrap the create-domain view which the old UI went through before showing an editor created thereby.  I'll remove it.